src/arrayMethodSort.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+// const compar = (a, b) => {
+//   const first = (a).toString();
+//   const second = (b).toString();
+
+//   if (first > second) {
+//     return 1;
+//   } else if (first === second) {
+//     return 0;
+//   } else {
+//     return -1;
+//   }
+// };
+const compar = (a, b) => {
```

Comment:
what is compar? Do you speak romanian?)) 
![image](https://user-images.githubusercontent.com/41700032/89903214-0d4cea80-dbf0-11ea-8fd5-a32ab27e4c3d.png)

Specialize names. stringComparator, compareLikeAString ex